docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,467 @@
+| New groups should be crated with a fixed frequency of relay request (every
+  `N`-th relay entry request should result in the creation of a new group).
+
+| Group creation
+| If according to group creation frequency the relay entry request is the one
+  that should trigger new group creation, the creation should be triggered once
+  a new relay entry appears on the chain (all off-chain clients should start the
+  process of creation of new group using the new entry value and a view
+  sortition pool function call).
+
+| Group creation
+| Frequency must be rare enough to leave the time for the group creation and
+  then (when the pool gets unlocked) for joining operators to pools.
+
+| Group creation
+| Group creation start transaction should be embedded into relay entry request
+  transaction.
+
+| Group creation
+| Group creation start transaction should lock the sortition pool and emit the
+  group creation start event.
+
+| Group creation
+| Group creation start transaction should be as cheap as possible.
+
+| Group creation
+| The sortition pool should weigh operators by stake and allow to select the
+  same operator to group multiple times.
+
+| Group creation - genesis
+| Beacon genesis should trigger the first group creation based on a fixed,
+  arbitrary seed value.
+
+| Group creation - genesis
+| It should not be possible to perform beacon genesis if they are some active
+  groups.
+
+| Group creation - genesis
+| Everybody should be able to run beacon genesis in the following situations: +
+  - when no groups were created by the random beacon before +
+  or +
+  - when there were some groups created by the random beacon before, but they
+  all have expired.
+
+| Group creation
+| Group creation start transaction should result in: +
+  - selection of a candidate group consisting of `group_size` members of
+  sortition pool +
+  - triggering of the DKG (distributed key generation) +
+  - the beacon should start the period of DKG result submission eligibility.
+
+| Croup creation - selecting members
+| Upon noticing the group creation start event, clients should choose the
+  sortition pool members to the candidate group based on the current relay
+  entry, ensuring that the higher is the stake of an operator, the higher is his
+  chance of being selected to the group.
+
+| Group creation - DKG
+| After group members are determined, they should perform off-chain DKG,
+  resulting either in success or timeout. The success result should contain list
+  of members of the candidate group, the public key of the group and list of
+  misbehaving members (members who were inactive or were disqualified) and
+  should be submitted by eligible group member on-chain.
+
+| Group creation - DKG submission
+| When determining validity of the tx submitting the DKG result, the following
+  aspects should be taken into account: +
+  - is the submitter eligible at the moment to send the tx? +
+  - are there enough supporting signatures on the result? +
```

Comment:
...and all those signatures are valid.

Reply:
Added, see 74353bb0ffc7205f4059cb08a6b0535cafd5e3c6.